integration-tests/retry.go

```diff
@@ -0,0 +1,26 @@
+package integrationtests
+
+import (
+	"time"
+)
+
+const (
+	max_retries     = 5
+	retry_wait_time = 3 * time.Second
+)
+
+type retryable = func() (string, error)
+
+// Simple retry in loop for commands produce only one string output and error
+func retry(f retryable) (output string, err error) {
+	for i := 0; i < max_retries; i++ {
+		output, err = f()
+		if err == nil {
+			return output, nil
+		} else {
+			time.Sleep(retry_wait_time)
+		}
```

Comment:
[nitpick] The last loop will wait for 3 seconds before finally exiting, we could add a small if condition here, (though I'm sure you can come up with a nicer and cleaner way to do this).
```suggestion
		} else if i != max_retries - 1 {
			time.Sleep(retry_wait_time)
		}
```

Reply:
Looks fine for me.